install.sh

```diff
@@ -0,0 +1,48 @@
+# Constants
+VERSION="v0.4.2"
+SIMPLEX_CHAT="simplex-chat"
```

Comment:
Not sure this should be a constant - it's used in different contexts where entities coincidentally have the same name (org, repo, bin name).. Any one gets renamed the constant has to be replaced in some places but not the others. Maybe better to remove it and just hardcode in the `PLATFORM_BIN` and `URL` variables

Reply:
I changed the name of the variables so it makes more sense to make it constants.